examples/.gitignore

```diff
@@ -1,2 +1,2 @@
 **/imagenet_classes.txt*
-
+/*.jpg
```

Comment:
No comment, but I am curious about what this was trying to achieve, given my experience with git is limited?

Reply:
This will ignore the downloaded *.jpg files in the git repo such that they don't show up as untracked files.

Reply:
This is a specific sub-directory scope git ignore file, so this means "ignore any files matching the glob '*.jpg' in this directory."

Reply:
Got it. Thank you so much for the clarifications. I assumed it was something related to catdog.jpg surgery. I was interested in learning how that fix would look.